### PR TITLE
Add Pass Rate By Variant to test analysis page.

### PR DESCRIPTION
**Pull request by dgoodwin**
This will allow us to see trends by variant instead of just by job
previously.

### CODE REVIEW

File: sippy-ng/src/tests/TestAnalysis.js [hunk 1]
```diff
@@ -204,8 +268,33 @@ export function TestAnalysis(props) {
   })
 
   let index = 0
+  variants.forEach((variant) => {
+    byVariantChart.datasets.push({
```
Comment (xueqzhan) [inline sippy-ng/src/tests/TestAnalysis.js:272]: Can this be put into the same loop on line 258?

Comment (dgoodwin) [reply sippy-ng/src/tests/TestAnalysis.js:270]: it can, good idea. Updated.

**Comment by dgoodwin** [comment]
Updated with unit test fix.

**Pull request merged**